CondTools/RunInfo/src/LHCInfoCombined.cc

```diff
@@ -0,0 +1,51 @@
+#include "CondTools/RunInfo/interface/LHCInfoCombined.h"
+
+LHCInfoCombined::LHCInfoCombined(const LHCInfo& lhcInfo) { setFromLHCInfo(lhcInfo); }
```

Comment:
the namespace should be lowercase, see 2.7 https://cms-sw.github.io/cms_coding_rules.html

Reply:
I'm not defining/using any namespace here - LHCInfoCombined is a class